torch/quantization/quantize.py

```diff
@@ -123,25 +125,27 @@ def add_quant_dequant(module):
         module._modules[name] = add_quant_dequant(child)
     return module
 
-def prepare(module, qconfig_dict=None):
-    r"""Prepares the module for calibration or training given a qconfig_dict.
-    Note that the module will be modified inplace but in case the input module
-    is a leaf module, a wrapped module will be returned.
+def prepare(model, qconfig_dict=None):
+    r"""Prepares the model for calibration or training given a qconfig_dict.
+    Note that the model will be modified inplace but in case the input model
+    is a leaf model, a wrapped model will be returned.
 
     Args:
-        mod: input module
+        mod: input model
         qconfig_dict: dictionary that maps from name of submodule to quantization
                       configuration
     Return:
-        A module with qconfig propogated, observer and quant dequant or fake
-        quant modules attached, a module that is ready for calibration or
+        A model with qconfig propogated, observer and quant dequant or fake
+        quant modules attached, a model that is ready for calibration or
         training
     """
-    propagate_qconfig(module, qconfig_dict)
+    propagate_qconfig(model, qconfig_dict)
     if qconfig_dict:
-        module = add_quant_dequant(module)
-    add_observer(module)
-    return module
+        model = add_quant_dequant(model)
+    add_observer(model)
```

Comment:
Lets call it as add_quantization_module(model) instead. Calling it observer is confusing.

Reply:
we need a name to cover both observer and fake_quant